localhost_apiserver/__init__.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+
+from flask import Flask, Response, request
+
+
+def create_app() -> Flask:
+    # create and configure the app
+    app = Flask(__name__, instance_relative_config=False)
+    app.config.from_mapping(SECRET_KEY="dev")
+
+    # read predefined mesh test response from a file
+    response_file_path = os.getenv("RESPONSE_FILE_PATH")
+    if not response_file_path:
+        app.logger.critical('Environment variable "RESPONSE_FILE_PATH" is required')
+        sys.exit(1)
+    app.logger.info(f'Serving mesh test results from "{response_file_path}"')
+    with open(response_file_path, mode="r") as f:
+        http_response_body = f.read()
+
+    # configure routes
+    @app.route("/synthetics/v202101beta1/health/tests", methods=["GET", "POST"])
+    def health_tests():
+        return Response(response=http_response_body, mimetype="application/json")
+
+    @app.route("/shutdown", methods=["GET"])
+    def shutdown():
```

Comment:
OK ... but is it really needed? The stub server can be stopped via docker command the same way as the webapp, right?

Reply:
> OK ... but is it really needed? The stub server can be stopped via docker command the same way as the webapp, right?

The stub server is not dockerized, I think we don't need it dockerized. Or do we...?

Reply:
Ah, I see. Dockerizing it would surely make it more reusable (especially if the map of requests and responses could be provided via a config), but we do not need to do it now. The current solution is fine for the intended purpose.

Reply:
I think we will basically improve the stub server once we have such a need.